src/styles/Hero.css

```
@@ -287,39 +294,69 @@ Link{
-  .hero-images{
-    margin-top: 170px;
+  
+  .countdown-number{
+    color: #ed444b;
+    font-size:10vw;
+ 
   }
+  
+  .countdown p{
+  
+  }
```

Biome (1.9.4)

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)